spec/presenters/openid_connect_user_info_presenter_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe OpenidConnectUserInfoPresenter do
+  include Rails.application.routes.url_helpers
+
+  let(:session_uuid) { SecureRandom.hex }
```

Comment:
Why not `SecureRandom.uuid`? Is there something meaningful about the random string format?